### PR TITLE
Add proxymode=userpsace to lxd deployments

### PR DESCRIPTION
**Pull request by ktsakalozos**

### CODE REVIEW

File: integration-tests/utils.py [hunk 1]
```diff
@@ -244,15 +244,20 @@ def all_units_ready(model):
...
-                data = yaml.load(f)
             for app in ['kubernetes-master', 'kubernetes-worker']:
                 options = data['services'][app].setdefault('options', {})
                 options['channel'] = snap_channel
-            with open(data_path, 'w') as f:
-                yaml.dump(data, f)
+        if await is_localhost():
+            if 'options' not in data['services']['kubernetes-worker']:
+                data['services'][app].setdefault('options', {})
+            options = data['services'][app]['options']
```
Comment (Cynerva) [inline integration-tests/utils.py]: Where does `app` come from? It looks like it comes from the `for` loop above, which is confusing, to say the least.

**Comment by Cynerva** [comment]
Is this blocking testing for the release? I would normally propose that we fix this in the charms or LXD profile instead, but I don't want to hold up a release for that.

File: integration-tests/utils.py (hunk 1 above)
Comment (ktsakalozos) [reply integration-tests/utils.py]: You are right, it's kubernetes-worker

**Comment by ktsakalozos** [comment]
Still running a couple of tests on lxd. I would not consider this blocking since we can ignore it for the release. This issue seems to be first reported https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/286 some time ago. I am +1 for fixing this in a more generic place but I'd rather not be in the charm since this is controller specific.

**Comment by Cynerva** [comment]
Ok. I would request that we leave this out of the integration tests, then. If LXD is broken then the tests should reflect it.

**Pull request closed without merging**